tests/pytests/test_cassini_drivers.py

```diff
@@ -140,7 +140,7 @@ def test_custom_frame_chain_iak(self, from_spice):
             target_frame_id.return_value = -800
             frame_chain = self.driver.frame_chain
             assert len(frame_chain.nodes()) == 0
-            from_spice.assert_called_with(center_ephemeris_time=2.4, ephemeris_times=[2.4], nadir=False, sensor_frame=14082360, target_frame=-800, exact_ck_times=True)
+            from_spice.assert_called_with(center_ephemeris_time=2.4, ephemeris_times=[2.4], nadir=False, sensor_frame=14082360, target_frame=-800, exact_ck_times=True,  inst_time_bias=0)
```

Comment:
Why is this only necessary on cassini drivers?

Reply:
Cassini has a special additional frame that is added to the end of the frame chain. It seems like that driver is the only thing testing the from_spice call.

I think that the NaifData class tests should have caught this as well, it's just hard to get all the checks.